packages/debian/metalk8s-sosreport-deb-2.4.0/debian/metalk8s-sosreport-deb.substvars

```diff
@@ -0,0 +1,2 @@
+misc:Depends=
+misc:Pre-Depends=
```

Comment:
Don't wee need to depends on some version of python and `sos`?

Reply:
This is `substvars`, we don't actually need this file to have contents (or at all???) if we fill up what we need in `control`.